dojo/product_type/views.py

```diff
@@ -22,70 +24,99 @@
 
 
 def product_type(request):
-    initial_queryset = Product_Type.objects.all().order_by('name')
-    name_words = [product.name for product in
-                  initial_queryset]
+    initial_queryset = Product_Type.objects.all().order_by("name")
```

Comment:
As recommended by PEP- 498, let's keep single quotes, unless double quotes are required.
```suggestion
    initial_queryset = Product_Type.objects.all().order_by('name')
```